src/configstore.ts

```diff
@@ -59,6 +66,16 @@ function getSettingsToAdd(config: any, label?: string, prefix?: string, tags?: T
     return settings;
 }
 
+function getSettingValue(value: any): string | undefined {
+    if (value === null || value === undefined) {
+        return undefined;
+    } else if (typeof value === "object") {

```

Comment:
>object [](start = 33, length = 6)

how about array, what we expect the target string?

Reply:
Note: For the value to be array at this point, a max flattening depth would have been specified to prevent the default flattening which splitting each array element into a separate key-value.

The string will look like "[1,2,..]" (like what the CLI import does when there is a max flattening depth)